web_services/foxx/brightid5/initdb.js

```diff
@@ -0,0 +1,108 @@
+const { query, db } = require('@arangodb');
+const collections = {
+  'connections': 'edge',
+  'groups': 'document',
+  'usersInGroups': 'edge',
+  'users': 'document',
+  'contexts': 'document',
+  'apps': 'document',
+  'sponsorships': 'edge',
+  'operations': 'document',
+  'operationsHashes': 'document',
+  'invitations': 'edge',
+  'variables': 'document',
+};
+
+// deprecated collections should be added to this array after releasing
+// second update to allow 2 last released versions work together 
+const deprecated = [
+  'removed',
+  'newGroups',
+  'usersInNewGroups',
+];
+
+function createCollections() {
+  console.log("creating collections if they don't exist ...");
+  for (let collection in collections) {
+    const coll = db._collection(collection);
+    if (coll) {
+      console.log(`${collection} exists`);
+    } else {
+      const type = collections[collection];
+      db._create(collection, {}, type);
+      console.log(`${collection} created with type ${type}`);
+    }
+  };
+}
+
+function removeDeprecatedCollections() {
+  console.log("removing deprecated collections");
+  for (let collection of deprecated) {
+    const coll = db._collection(collection);
+    if (coll) {
+      db._drop(collection);
+      console.log(`${collection} dropped`);
+    } else {
+      console.log(`${collection} dropped before`);
+    }
+
+  }
+}
+
+function v5() {
+  const contextsColl = db._collection('contexts');
+  const appsColl = db._collection('apps');
+  const contexts = contextsColl.all().toArray();
+  for (let context of contexts) {
+    appsColl.insert({
+      _key: context['_key'],
+      name: context['_key'],
+      context: context['_key'],
+      url: context['appUrl'],
+      logo: context['appLogo'],
+      totalSponsorships: context['totalSponsorships'],
+      sponsorPublicKey: context['sponsorPublicKey'],
+      sponsorPrivateKey: context['sponsorPrivateKey'],
+      contractAddress: context['sponsorPrivateKey'],
```

Comment:
First, this could be named something that makes it more obvious what it is, like "sponsorEventContract".  Second, why is the initial value being set to `sponsorPrivateKey`?

Reply:
First one applied.
Second one was a mistype that solved.
:pray: